ChangeLog

```diff
@@ -18,6 +19,7 @@
  * Removed deprecated GramSchmidtAlgorithm, ChebychevAlgorithm classes
  * Removed [gs]etOptimizationSolver methods
  * Removed CovarianceModel::compute{AsScalar,StandardRepresentative} overloads
+ * Deprecated MonteCarlo, ImportanceSampling, QuasiMonteCarlo, RandomizedQuasiMonteCarlo, RandomizedLHS classes
 
```

Comment:
"Note that the LHS class is kept, as it's not currently used as a sequential algorithm as the total outersampling must be known to compute the shuffle matrix. This is potentially ambiguous as one may also use ProbabilitySimulation with LHSExperiment to get the behavior of the old RandomizedLHS class."